### DISCUSSION TITLE
move window

### DISCUSSION THREAD

**Discussion by aloispichler** [hyprwm/Hyprland#3257] [General]
I have
`bind= $mainMod SHIFT, left, movewindow, l`,
`bind= $mainMod SHIFT, right, movewindow, r`., etc. to move a window on the screen.

Suppows we have 3 windows
```
|Focus |      |
 ------|  2   |
|   3  |      |
```
then `$mainMod SHIFT right` results in
```
|  2   |      |
 ------|Focus |
|   3  |      |
```
as desired.

But `$mainMod SHIFT left` should result in
```
|      |   2   |
|Focus | ------
|      |   3   |
```
This is at least what I am used from i3.
How/ can this be accomplished?

**Comment by MightyPlaza** [comment]
checkout hy3

**Reply by aloispichler** [reply]
I have found hy3-git on AUR, but this requires manual intervention…

**Reply by MightyPlaza** [reply]
https://github.com/outfoxxed/hy3

**Reply by aloispichler** [reply]
that's what I did in arch, this is what requires manual intervention.

**Reply by MightyPlaza** [reply]
wdym manual intervention?

**Reply by aloispichler** [reply]
I am running `yay -Syu hy3-git` (yay is for pacman in aur),
and the final message is
==> ERROR: A failure occurred in build().
Aborting...
-> error making: hy3-git-exit status 4
-> Failed to install the following packages. Manual intervention is required:
hy3-git - exit status 4

**Reply by MightyPlaza** [reply]
that mean installing failed
follow the manual build instructions on the github repo

**Comment by aloispichler** [comment]
any other, simpler ideas than hy3?

**Comment by StoicaTielemans** [comment]
i use this
bind = $mainMod SHIFT, left, swapwindow, l 
bind = $mainMod SHIFT, right, swapwindow, r 
bind = $mainMod SHIFT, up, swapwindow, u 
bind = $mainMod SHIFT, down, swapwindow, d
